instrumentation/oshi/oshi-doc.md

```diff
@@ -0,0 +1,53 @@
+
+OSHI Jar Setup with Special Configuration
+
+Introduction: OSHI (Operating System Hardware Information) is a Java library that provides cross-platform access to information about system hardware, operating system, and JVM. Sometimes, you may need to manually download the OSHI jar file and add it to the system classpath, especially in environments where dependency management tools like Maven or Gradle are not available or suitable.
+
+
+Downloading OSHI Jar: You can download the OSHI jar file from the official Maven repository or from the GitHub releases page.
+
+1.	From Maven Repository: You can download the OSHI jar directly from the Maven repository using a web browser or a command-line tool like wget or curl.
+
+Cmd:
+wget https://repo1.maven.org/maven2/com/github/oshi/oshi-core/v1/oshi-core-v1.jar 
+
+Note: In above cmd, v1 denotes the version for the .jar
+
+
+
+
+2.	From GitHub Releases: OSHI releases are also available on GitHub. You can download the jar file from the releases page.
+•	Go to the OSHI GitHub releases page: OSHI Releases
+•	Download the desired version of the oshi-core jar file.
+
+Adding OSHI Jar to Classpath:
+
+Once you've downloaded the OSHI jar file, you need to add it to the classpath of your Java application.
+
+1. Command Line: You can add the OSHI jar to the classpath using the -cp or -classpath option when running the java command.
+
+Cmd:
+java -cp /path/to/oshi-core-v1.jar MainClass 
+
+Note: In above cmd, v1 denotes the version for the .jar
+
+Replace /path/to/oshi-core-v1.jar with the actual path to the downloaded jar file and MainClass with the main class of your Java application.
+
+2. Environment Variable: Set the CLASSPATH environment variable to include the path to the OSHI jar file.
+
+Cmd:
+export CLASSPATH=/path/to/oshi-core-v1.jar:$CLASSPATH 
```

Comment:
instructions like this are problematic because the path separator is different on win and nix